src/main.rs

```diff
@@ -1,52 +1,94 @@
 extern crate docopt;
 extern crate rand;
 extern crate rustc_serialize;
-#[macro_use] extern crate lazy_static;
 
-use std::process;
+use std::{process, io, path};
+use std::fs::File;
+use std::io::Read;
 use rand::distributions::{IndependentSample, Range};
 use docopt::Docopt;
 
 const USAGE: &'static str = "
 Generate XKCD style passwords.
 
 Usage:
-  xkcdpass [-c <count>]
+  xkcdpass [options] [-w <list>...]
```

Comment:
Could you keep the `[-c <count>]` in there directly instead of replacing it with `[options]`? I think it makes it easier to read.